### DISCUSSION TITLE
How to configure RIPGREP_CONFIG_PATH for PowerShell?

### DISCUSSION THREAD

**Discussion by heitorsilva** [BurntSushi/ripgrep#2370] [Q&A] [answered]
Hello!

I'm running Windows 11, using the native Terminal, with PowerShell 7.3.0.

I have ripgrep 13.0.0 (rev af6b6c543b)  installed via chocolatey 1.2.1.

I tried setting RIPGREP_CONFIG_PATH via Windows System-wide environment variables...
I tried setting RIPGREP_CONFIG_PATH directly on PowerShell and then running rg...

None of the above made ripgrep recognize my config file.

How can I make rg read my RIPGREP_CONFIG_PATH, or, how can I pass a config file to it on PowerShell?

**Comment by heitorsilva** [comment, marked as answer]
I solved my issue by setting user variables for my user, and instead of using `$HOME\.ripgreprc` in the value, I used the full path, like: `C:\Users\blabla\.ripgreprc`

**Reply by BurntSushi** [reply]
Thank you for sharing your answer!
